invenio_app_ils/indexer.py

```diff
@@ -189,10 +189,20 @@ def _prepare_record(self, record, index, doc_type):
         data = record.dumps()
         return data
 
+    def _prefix_index(self, app, index):
+        """Prefixes the given index if needed.
+        :param app: Flask app to get the config from.
+        :param index: Name of the index to prefix.
+        :returns: A string with the new index name prefixed if needed.
+        """
+        index_prefix = app.config['SEARCH_INDEX_PREFIX'] or ''
```

Comment:
maybe we should add a line with that config variable with description to let people know it is possible to configure it ? WDYT?

Reply:
this is coming from invenio-indexer but I will add a line pointing there for ref :)